src/Playwright/Core/BrowserType.cs

```diff
@@ -119,6 +119,16 @@ public async Task<IBrowserContext> LaunchPersistentContextAsync(string userDataD
                 ignoreAllDefaultArgs: options.IgnoreAllDefaultArgs,
                 baseUrl: options.BaseURL,
                 forcedColors: options.ForcedColors).ConfigureAwait(false)).Object;
+
+            // TODO: unite with a single browser context options type which is derived from channels
+            context.Options = new()
```

Comment:
this needs a proper refactor, so we can share the same underlying types between NewContext and LaunchPersistentContext. For now just manually wrapping the values over.